test/boost/encryption_at_rest_test.cc

```diff
@@ -949,6 +960,7 @@ SEASTAR_TEST_CASE(test_gcp_provider_with_master_key_in_cf, *check_run_test_decor
         );
 
         // should fail
+        // fixme: Catch the exact exception we expect to be raised here
```

Comment:
catch-all pattern seems to be all over this file. Do you plan to tackle them all here or one by one?

Reply:
This is the last one, did you find anything else?